bench/journeys_advisory.go

```
@@ -0,0 +1,348 @@
+	lower := strings.ToLower(observation.Stdout)
+	for _, forbidden := range []string{"receipt", "delivery", "\"pass\""} {
+		if strings.Contains(lower, forbidden) {
+			return fmt.Errorf("advisory validate output claims authority (%q): %s", forbidden, observation.Stdout)
+		}
+	}
+	var validated struct {
+		TransportValidated bool `json:"transport_validated"`
+		FindingCount       int  `json:"finding_count"`
+	}
+	if err := json.Unmarshal([]byte(strings.TrimSpace(observation.Stdout)), &validated); err != nil || !validated.TransportValidated {
+		return fmt.Errorf("advisory validate did not admit the clean result (err=%v): %s", err, observation.Stdout)
```

golangci-lint (2.12.2)

[error] non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)